src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -690,41 +411,25 @@
       task will require the outputs of `proguard.csproj`.
       -->
     <ProjectReference Include="..\proguard\proguard.csproj">
-      <Project>{4B9D96BB-95AB-44E8-9F87-13B12C8BCED1}</Project>
-      <Name>proguard</Name>
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
     </ProjectReference>
     <ProjectReference Include="..\r8\r8.csproj">
-      <Project>{1bafa0cc-0377-46ce-ab7b-7bb2e7b62f63}</Project>
-      <Name>r8</Name>
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
     </ProjectReference>
     <ProjectReference Include="..\aapt2\aapt2.csproj">
-      <Project>{0c31de30-f9df-4312-bffe-dcad558ccf08}</Project>
-      <Name>aapt2</Name>
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
     </ProjectReference>
     <ProjectReference Include="..\bundletool\bundletool.csproj">
-      <Project>{a0aef446-3368-4591-9de6-bc3b2b33337d}</Project>
-      <Name>bundletool</Name>
-      <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
-    </ProjectReference>
-    <ProjectReference Include="..\..\external\xamarin-android-tools\src\Xamarin.Android.Tools.AndroidSdk\Xamarin.Android.Tools.AndroidSdk.csproj">
-      <Project>{E34BCFA0-CAA4-412C-AA1C-75DB8D67D157}</Project>
-      <Name>Xamarin.Android.Tools.AndroidSdk</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\external\Java.Interop\src\Java.Interop.Export\Java.Interop.Export.csproj">
-      <Project>{B501D075-6183-4E1D-92C9-F7B5002475B1}</Project>
-      <Name>Java.Interop.Export</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\external\Java.Interop\src\Java.Interop\Java.Interop.csproj">
-      <Project>{94BD81F7-B06F-4295-9636-F8A3B6BDC762}</Project>
-      <Name>Java.Interop</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\manifestmerger\manifestmerger.csproj">
-      <Project>{af8ac493-40ac-4195-82f6-b08ee4b4e49e}</Project>
-      <Name>manifestmerger</Name>
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
     </ProjectReference>
+    <ProjectReference Include="..\..\external\xamarin-android-tools\src\Xamarin.Android.Tools.AndroidSdk\Xamarin.Android.Tools.AndroidSdk.csproj" />
+    <ProjectReference Include="..\..\external\Java.Interop\src\Java.Interop.Export\Java.Interop.Export.csproj" />
+    <ProjectReference Include="..\..\external\Java.Interop\src\Java.Interop\Java.Interop.csproj" />
   </ItemGroup>
+
+  <!-- Import Microsoft.NET.Sdk targets before our targets so we can override behavior -->
+  <!-- See https://github.com/microsoft/msbuild/pull/4922 -->
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
```

Comment:
On previous projects where I have had to do this I only needed the `Sdk.targets` import. so the `Sdk.props` one *might* be redundant.